Source/CarthageKit/Archive.swift

```diff
@@ -13,7 +13,7 @@ import ReactiveTask
 
 /// Zips the given input items (recursively) into an archive that will be
 /// located at the given URL.
-public func zipIntoArchive(destinationArchiveURL: NSURL, workingDirectory: String, inputPaths: [String]) -> SignalProducer<(), CarthageError> {
+public func zipArchive(into destinationArchiveURL: NSURL, workingDirectory: String, inputPaths: [String]) -> SignalProducer<(), CarthageError> {
```

Comment:
This doesn't seem quite right. It's not zipping the archive itself; it's zipping into it.

Maybe we should rearrange things like so?

```
zip(paths: [String], into archiveURL: NSURL, workingDirectory: String)
```